specification/reservations/resource-manager/Microsoft.Capacity/preview/2020-10-01-preview/examples/ChangeDirectoryReservationOrder.json

```diff
@@ -0,0 +1,29 @@
+{
+  "parameters": {
+    "api-version": "2020-10-01-preview",
+    "reservationOrderId": "a075419f-44cc-497f-b68a-14ee811d48b9",
+    "body": {
+      "destinationTenantId": "906655ea-30be-4587-9d12-b50e077b0f32"
+    }
+  },
+  "responses": {
+    "200": {
+      "body": {
+        "reservationOrder" : {
+          "id": "1f14354c-dc12-4c8d-8090-6f295a3a34aa",
```

Comment:
should this match the id in line 4?

Reply:
Will correct this to match id in this example.